pages/library/add.js

```diff
@@ -141,51 +81,145 @@ export default function AddWisdom({
     ).length;
     const itemsInInventory = userData[0].books[userBookIndex].inventory;
 
-    if (wisdomsInBook % 20 === 0 && gainedItems < wisdomsInBook / 20) {
+    if (wisdomsInBook % 5 === 0 && gainedItems < wisdomsInBook / 5) {
```

Comment:
Why 5?

Constants that are not self explanatory (like 0 or 1) should be better defined as a constant somewhere, e.g.:

```js
const MAX_WISDOMS = 5;

// ...

if (wisdomsInBook % MAX_WISDOMS === 0 && gainedItems < wisdomsInBook / MAX_WISDOMS)  { ... }
```